### PR TITLE
[BPKR-39] Added new smiley icons face--blank, face--happy, face--sad.

### PR DESCRIPTION
**Pull request by ojcurtis**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 > Place your changes below this line.
 
+# 9.3.2
```
Comment (shaundon) [inline UNRELEASED.md]: Will be `9.4.0` as we're adding things.

Comment (ojcurtis) [reply UNRELEASED.md]: Done!

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:54e04beed3d1fd667907369cf8df56835f3dc71c`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:bcdcbdf8390e29316688470297dfbd24f4048066`

**Pull request merged**